website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -40,6 +40,13 @@ Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt
 
 2. Save the `dbt_cloud.yml` file in the `.dbt` directory, which stores your dbt Cloud CLI configuration. Store it in a safe place as it contains API keys. Check out the [FAQs](#faqs) to learn how to create a `.dbt` directory and move the `dbt_cloud.yml` file.
    
+    - North America: https://YOUR_ACCESS_URL/cloud-cli
```

Comment:
i know we're in beta but should we also add the azure mt? we might need to think about a snippet or easier approach to the URLs maybe for the future?